sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -2127,5 +2127,36 @@ ALTER TABLE `form_bronchitis` MODIFY `diagnosis3_bronchitis_form` text;
 ALTER TABLE `form_bronchitis` MODIFY `diagnosis4_bronchitis_form` text;
 #EndIf
 
+#IfColumn patient_data care_team
+ALTER TABLE `patient_data` DROP COLUMN `care_team`;
+#EndIf
+
+#IfMissingColumn patient_data care_team_provider
+ALTER TABLE `patient_data` ADD COLUMN `care_team_provider` int(11) DEFAULT NULL;
+#EndIf
+
+#IfMissingColumn patient_data care_team_facility
+ALTER TABLE `patient_data` ADD COLUMN `care_team_facility` int(11) DEFAULT NULL;
+#EndIf
+
```

Comment:
Would do a query outside a if block that simply changes care_team field_id to care_team_provider

Reply:
actually, see below post for a better plan